frontends/p4/fromv1.0/converters.h

```diff
@@ -78,12 +78,23 @@ class TypeConverter : public ExpressionConverter {
     explicit TypeConverter(ProgramStructure* structure) : ExpressionConverter(structure) {}
 };
 
+class ExternConverter {
```

Comment:
I am guessing you are subclassing this somewhere. In this case, I think this should be an interface (abstract class), and there should be also a "default" implementation.